src/domain/newsletter/__tests__/SubscribeNewsletterPage.test.tsx

```diff
@@ -73,11 +73,7 @@ describe('SubscribeNewsletterPage', () => {
 
     await waitFor(
       () => {
-        expect(
-          screen.getByRole('heading', {
-            name: /tilaus lähetetty/i,
-          })
-        ).toBeInTheDocument();
+        expect(screen.getAllByText(/tilaus lähetetty/i)).toHaveLength(1);
```

Comment:
This should be reverted. I tested with the old implementation and it's working ok

Reply:
actually, I tried with `GetByText`

Reply:
fixed